WebRoot/public/login.jsp

```diff
@@ -699,7 +703,7 @@ if (application.getInitParameter("offlineMode") != null) {
                                             <select id="client" name="client" onchange="clientChange(this.options[this.selectedIndex].value)">
                                                 <option value="preferred" <c:if test="${client eq 'preferred'}">selected</c:if> > <fmt:message key="clientPreferred"/></option>
                                                 <option value="advanced" <c:if test="${client eq 'advanced'}">selected</c:if>> <fmt:message key="clientAdvanced"/></option>
-                                                <c:if test="${modernSupported}">
+                                                <c:if test="${modernSupported and !(ua.isModernIE)}">
```

Comment:
we can just update modernSupported condition to be false when user is using IE11, rest everything is handled I guess

Reply:
Fixed.